ChangeLog

```diff
@@ -32,7 +32,7 @@
 === Miscellaneous ===
 
 === Bug fixes ===
-
+ * #924 (Probleme with the factory of a Generalized Pareto distribution)
```

Comment:
could you move this to the 1.10 section ? It will be easier for me to backport to the 1.10 branch ?

Reply:
@jschueller You mean, the 1.10rc1 section? Or do I have to create a new 1.10 section? There are currently only two sections: 1.11 and 1.10rc1.

Reply:
Yes, 1.10rc1